src/FormCheck.tsx

```diff
@@ -176,9 +148,9 @@ const FormCheck: FormCheck = (React.forwardRef(
           style={style}
           className={classNames(
             className,
-            bsPrefix,
-            custom && `custom-${type}`,
+            label && bsPrefix,
             inline && `${bsPrefix}-inline`,
+            type === 'switch' && 'form-switch',
```

Comment:
Should we add a bsPrefix override for `form-switch`?

Reply:
maybe we should treat `<Switch>` as a separate component that passes down its own `bsPrefix` that defaults to `form-switch`?

Reply:
The switch still needs the `.form-check` class alongside `.form-switch`

ie.  `<div class="form-check form-switch">`

I guess users would need ability to control both.

Reply:
ah, so like `switchBsPrefix` then?

Reply:
Yeah.  Wanted to double check before I added something like that :)